scratch/app-zoom.jsonc

```diff
@@ -0,0 +1,73 @@
+{
+  "type": "view",
+  "bindings": [
+    {
+      "type": "view",
+      "bindings": [
+        {
+          "type": "markdown",
+          "label": "##### Your Personal Meeting ID\n440 287 4306\nhttps://mattermost.zoom.us/my/matt.birtch"
+        },
+        {
+          "type": "button",
+          "label": "Start a Meeting",
+          "icon": "icons/zoom-icon-small.png",
+          "submit": {
+            "path": "actions/start_meeting",
+            "state": {
+              "meeting_id": "4402874306"
+            }
+          }
+        }
+      ]
+    },
+    {
+      "type": "divider"
+    },
+    {
+      "type": "markdown",
+      "label": "Upcoming Zoom Meetings"
+    },
+    {
+      "type": "view",
+      "bindings": [
+        {
+          "type": "markdown",
+          "label": "Customer Obsession Meeting\n11:00AM - 12:00PM\nHappening right now"
```

Comment:
Note that I went with the more manual approach here, instead of using a List Block. This is because the content for the list items in the Zoom app design are more complex than could be defined by one List Item element, using the nested data model structure supported by nested bindings.